content/en/docs/concepts/workloads/pods/_index.md

```diff
@@ -14,12 +14,11 @@ card:
 
 _Pods_ are the smallest deployable units of computing that you can create and manage in Kubernetes.
 
-A _Pod_ (as in a pod of whales or pea pod) is a group of one or more
-{{< glossary_tooltip text="containers" term_id="container" >}}, with shared storage and network resources, and a specification for how to run the containers. A Pod's contents are always co-located and
+A _Pod_ (as in a pod of whales or pea pod) is a cluster of
+{{< glossary_tooltip text="containers" term_id="containers" >}}, that shares storage and network resources, with a guideline on how to run these containers. A Pod's contents are always co-located and
```

Comment:
Should we wrap the long lines as we did for all other paragraphs?

Reply:
What do you mean by "guideline"?

Reply:
Did the wrapping part.
I thought "guideline" would be more suited, but when dug more into the context decided to revert it back to "specification"